client-ts/signalr-protocol-msgpack/spec/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+    "compileOnSave": false,
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es5",
+        "moduleResolution": "node",
+        "outDir": "./obj",
+        "lib": [ "es2015", "dom" ],
+        "baseUrl": ".",
+        "paths": {
+            "@aspnet/signalr": [ "../../signalr/src/index" ]
+        }
```

Comment:
This allows us to use `import {} from "@aspnet/signalr"` even though we don't actually use the `@aspnet/signalr` NPM package, we want to use it from source in the sibling directory.